ra_aid/agent_utils.py

```
@@ -510,10 +666,12 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         _CONTEXT_STACK.remove(self)
 
+
 def check_interrupt():
     if _CONTEXT_STACK and _INTERRUPT_CONTEXT is _CONTEXT_STACK[-1]:
         raise AgentInterrupt("Interrupt requested")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)